cve_bin_tool/cvedb.py

```diff
@@ -274,6 +274,21 @@ def init_database(self):
         cursor.execute(cve_data_create)
         cursor.execute(version_range_create)
         cursor.execute(index_range)
+
+        # Check that latest schema is being used
+        self.LOGGER.info("Check database is using latest schema")
+        schema_check = "SELECT * FROM cve_severity WHERE 1=0"
+        result = cursor.execute(schema_check)
+        latest_schema = False
+        # Look through column names and find column added in latest schema
+        for col_name in result.description:
+            if col_name[0] == "description":
+                latest_schema = True
+        if not latest_schema:
+            # Recreate table using latest schema
+            self.LOGGER.info("Upgrading database to latest schema")
+            cursor.execute("DROP TABLE cve_severity")
+            cursor.execute(cve_data_create)
```

Comment:
This won't work. You need to clear whole cache directory because we also changed cache format in new version. We are now storing cache of curl also. 
You should just call `self.clear_cached_data()` here.

Reply:
It'll work if you have a version between when we changed the format and when the severity table was added -- I don't think they happened at the same time.

That said, self.clear_cached_data() is probably the right choice if we want to future-proof this and make it work for between version updates, and I think we probably do.  We treat .cache as ephemeral pretty consistently in other places too so that it can be recreated at will from NVD.  (And now, any deviations from what's in NVD would be stored in triage data.  Previously you were just out of luck if you wanted that and would have to apply a script or something.)